test/functional/smart_proxy_auth_test.rb

```diff
@@ -63,6 +63,5 @@ def test_failed_authentication_with_invalid_basic_auth_credentials_in_web_ui_con
     get :externalNodes, :id => 123
 
     assert_response :forbidden
-    assert_template "common/403"
```

Comment:
reverting it does not add any failure and seems to work, I suppose we still render the template here since you didn't specify text format